packages/landing/src/styles/docs.css

```
@@ -0,0 +1,265 @@
+  padding-top: 1.75rem;
+}
+.right-sidebar-panel {
+  padding-top: 1.75rem;
+}
+.content-panel {
+  padding-top: 0.5rem;
+}
+
+/* Headings use Lora to match the landing logo/serif tone. */
+:root {
+  --sl-font-headings: "Lora", Georgia, serif;
```

Stylelint (17.13.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)